packages/components/src/components/contract-card/contract-card-items/contract-card-body.jsx

```diff
@@ -185,79 +55,106 @@ const ContractCardBody = ({
+                indicative={indicative}
+                is_sold={is_sold}
+                onMouseLeave={onMouseLeave}
+                status={status}
+                removeToast={removeToast}
+                setCurrentFocus={setCurrentFocus}
+                is_positions={is_positions}
+            />
+        );
+    } else {
+        card_body = (
+            <React.Fragment>
+                <div className='dc-contract-card-items-wrapper'>
+                    <ContractCardItem
+                        header={is_sold ? PROFIT_LOSS : POTENTIAL_PROFIT_LOSS}
+                        is_crypto={isCryptocurrency(currency)}
+                        is_loss={+profit < 0}
+                        is_won={+profit > 0}
                     >
-                        {status === 'profit' && <Icon icon='IcProfit' />}
-                        {status === 'loss' && <Icon icon='IcLoss' />}
-                    </div>
-                </ContractCardItem>
-                <ContractCardItem header={getCardLabels().PURCHASE_PRICE}>
-                    <Money amount={buy_price} currency={currency} />
-                </ContractCardItem>
-                <ContractCardItem header={getCardLabels().POTENTIAL_PAYOUT}>
-                    <Money currency={currency} amount={payout} />
-                </ContractCardItem>
-            </div>
-            <MobileWrapper>
-                <div className='dc-contract-card__status'>
-                    {is_sold ? (
-                        <ResultStatusIcon
-                            getCardLabels={getCardLabels}
-                            is_contract_won={getDisplayStatus(contract_info) === 'won'}
-                        />
-                    ) : (
-                        progress_slider_mobile_el
-                    )}
+                        <Money amount={profit} currency={currency} />
+                        <div
+                            className={classNames('dc-contract-card__indicative--movement', {
+                                'dc-contract-card__indicative--movement-complete': is_sold,
+                            })}
+                        >
+                            {status === 'profit' && <Icon icon='IcProfit' />}
+                            {status === 'loss' && <Icon icon='IcLoss' />}
+                        </div>
+                    </ContractCardItem>
+                    <ContractCardItem header={is_sold ? PAYOUT : INDICATIVE_PRICE}>
+                        <Money currency={currency} amount={sell_price || indicative} />
+                        <div
+                            className={classNames('dc-contract-card__indicative--movement', {
+                                'dc-contract-card__indicative--movement-complete': is_sold,
+                            })}
+                        >
+                            {status === 'profit' && <Icon icon='IcProfit' />}
+                            {status === 'loss' && <Icon icon='IcLoss' />}
```

Comment:
```suggestion
                            {<Icon icon={status === 'profit' ? 'IcProfit' : 'IcLoss' } />}
```

Reply:
unfortunately, status can be also null, that's why in such case it will show the icon for 'lost' 😞